ragstack-e2e-tests/e2e_tests/langchain/test_unstructured.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from langchain_community.document_loaders import UnstructuredAPIFileLoader
```

Comment:
I believe we should have a pinned version of the client in the pyproject.toml of ragstack. 
We should find the current range configured in langchain-community and pick a version